src/test/kotlin/com/github/imflog/schema/registry/compatibility/CompatibilityTaskActionTest.kt

```diff
@@ -30,10 +30,10 @@ class CompatibilityTaskActionTest {
         val parser = Schema.Parser()
         val testSchema = parser.parse("{\"type\": \"record\", \"name\": \"test\", \"fields\": [{ \"name\": \"name\", \"type\": \"string\" }]}")
         val registryClient = MockSchemaRegistryClient()
-        registryClient.register("test", testSchema)
+        registryClient.register("testcompatibility", testSchema)
```

Comment:
Not sure about what you changed in this file, could you please explain ?

Reply:
rolled back this change